rclcpp/include/rclcpp/experimental/executable_list.hpp

```diff
@@ -27,7 +27,7 @@
 
 namespace rclcpp
 {
-namespace executor
+namespace experimental
```

Comment:
Why `experimental` versus `detail`?

Reply:
Because I think it's completely redundant with new features related to the new wait set. I'd like to remove it completely, but haven't had time to do that yet.

I chose not to put it in detail, because I'm not sure about it (expect it might change) or not sure it will stick around. Whereas something in detail, for me, we don't expect it to change or be removed.